server/pom.xml

```diff
@@ -49,6 +49,7 @@
         <module>apps/scaling-pulsar-smtp</module>
         <module>apps/spring-app</module>
         <module>apps/webadmin-cli</module>
+        <module>apps/webadmin-http-client</module>
```

Comment:
I would prefer to have it in `server/protocols/webadmin/webadmin-http-client`. After all this is not an application (no Main)